### PR TITLE
Energy consumption plugin

### PR DESCRIPTION
**Pull request by delanoflipse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Cache files
 __pycache__/
 *.py[cod]
-*$py.class
\ No newline at end of file
+*$py.class
+.venv/
+.vscode/
```
Comment (aron-hoogeveen) [inline .gitignore:6]: This should not go into the public `.gitignore`, but instead go into your local `.git/info/exclude` or equivalent

File: README.md [hunk 2]
```diff
@@ -2,8 +2,56 @@
...
+
+## Description
+
+<!-- TODO add project description -->
+
+## Usage
+
+There are two main ways to use this project:
+
+1. Use the decorator to test a function. You can proved the amount of times it has to run as a parameter.
```
Comment (aron-hoogeveen) [inline README.md:18]: You probably mean "provide"?

**Comment by aron-hoogeveen** [comment]
If I am correct, there is also a lot of code/data from the [spec-power-model](https://github.com/green-coding-solutions/spec-power-model/). Therefore, the license must be changed to a new one ([the same](https://github.com/green-coding-solutions/spec-power-model/blob/main/LICENSE) as from that repo) and I think wherever there are changes to their code a notion should be included that the code is modified.

**Review by RodinHaker — DISMISSED** [review]

**Review by delanoflipse — COMMENTED** [review]
LGTM 🚀

**Review by aron-hoogeveen — APPROVED** [review]
Would not prefer the `.vscode/` exclude in `.gitignore` but will approve.

**Pull request merged**